packages/block-library/src/list-item/edit.js

```diff
@@ -42,14 +41,14 @@ function IndentUI( { clientId } ) {
 				title={ __( 'Outdent' ) }
 				describedBy={ __( 'Outdent list item' ) }
 				disabled={ ! canOutdent }
-				onClick={ outdentListItem }
+				onClick={ () => outdentListItem() }
```

Comment:
Why are we doing this change here? Can't we just pass outdentListItem as before?